core/station/api/spec.did

```diff
@@ -774,6 +774,12 @@ type CallExternalCanisterOperation = record {
   // The reply blob produced by a successful call of the execution method,
   // i.e., when the request is `Completed`.
   execution_method_reply : opt blob;
+  // Optional details about the call operation, this field is not populated in list responses, only
+  // when using `get_request` to avoid going over the response size limit.
+  with_details : opt record {
```

Comment:
why this additional layer instead of inlining arg : opt blob as a regular field of `CallExternalCanisterOperation`?

Reply:
The idea would be to make it very explicit that this is optional and is not a top level field of the main type, but maybe inlining arg would be good enough, wdyt?